src/normal.cc

```diff
@@ -1311,6 +1316,25 @@ void select_object(Context& context, NormalParams params)
             return;
         }
 
+        if (key == alt(';'))
+        {
+            String object_flags;
+            if (flags & ObjectFlags::Inner)
+                object_flags = "inner";
+            if (flags & ObjectFlags::ToBegin)
+                object_flags += format("{}to_begin", object_flags == "" ? "" : ",");
+            if (flags & ObjectFlags::ToEnd)
+                object_flags += format("{}to_end", object_flags == "" ? "" : ",");
+            EnvVarMap env_vars = {
+                { "count", to_string(params.count) },
+                { "register", String{&params.reg, 1} },
+                { "select_mode", mode == SelectMode::Extend ? "extend" : "replace" },
+                { "object_flags", object_flags }
```

Comment:
I was wondering between exposing those values like you did, or exposing the keystroke that led to them. The keystroke seems simpler, but I suspect in most use cases where you'd use that keystroke, you could just get it with a non-command based key mapping in object mode.